.github/workflows/auto-update.yml

```diff
@@ -0,0 +1,57 @@
+name: Update Graypaper
+
+on:
+  schedule:
+    - cron: "38 * * * *" # using an arbitrary minute to avoid high traffic
+
+jobs:
+  check_for_updates:
+    name: Check for updates
+    runs-on: ubuntu-latest
+    outputs:
+      local_version: ${{steps.local_version.outputs.VERSION}}
+      remote_version: ${{steps.remote_version.outputs.VERSION}}
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v4
+      with:
+        submodules: 'true'
+    - name: Check local version
+      id: local_version
+      run: echo "VERSION=$(cat graypaper/VERSION)" >> "$GITHUB_OUTPUT"
+    - name: Get latest graypaper and compare versions
```

Comment:
I'm wondering if it's possible to monitor new releases instead of `VERSION` file in the repo. But that's good enough too, releases would just be "more correct" :)

Reply:
possible for sure. ill see if the effort is reasonable

Reply:
id defer this to near future